sdk/communication/azure-communication-administration/README.md

```diff
@@ -262,21 +262,47 @@ The Phone Number Client supports a variety of long running operations that allow
 
 ### Create Search
 
-<!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L370-L380 -->
+<!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L354-L378 -->
 ```java
-        SyncPoller<PhoneNumberSearch, PhoneNumberSearch> res = 
-            phoneNumberClient.beginCreateSearch(createSearchOptions, duration);
-        res.waitForCompletion();
-        PhoneNumberSearch result = res.getFinalResult();
-
-        System.out.println("Search Id: " + result.getSearchId());
-        for (String phoneNumber: result.getPhoneNumbers()) {
-            System.out.println("Phone Number: " + phoneNumber);
-        }
-    }
+String phonePlanId = "PHONE_PLAN_ID";
+
+List<String> phonePlanIds = new ArrayList<>();
+phonePlanIds.add(phonePlanId);
+
+CreateSearchOptions createSearchOptions = new CreateSearchOptions();
+createSearchOptions
+    .setAreaCode("AREA_CODE_FOR_SEARCH")
+    .setDescription("DESCRIPTION_FOR_SEARCH")
+    .setDisplayName("NAME_FOR_SEARCH")
+    .setPhonePlanIds(phonePlanIds)
+    .setQuantity(2);
+
+Duration duration = Duration.ofSeconds(1);
```

Comment:
I know this was my comment in the initial PR but now that I see it in action, I feel that the pollInterval shouldn't be necessary to pass in.. I still believe that this should be an option but not mandated and in case it is not passed we can take in a default interval.. you probably don't need to do it in this PR, just take care of it in a separate one.. I am also curious to see how other languages are doing it..

Reply:
We can make an override function in the next PR to address this 👍🏻